### PR TITLE
Add support for beacon

### PR DESCRIPTION
**Pull request by mwrrlvkixj**

### CODE REVIEW

File: utils/deps.go [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+
 	"github.com/Zettablock/zsource/configs"
 	"golang.org/x/exp/slog"
 	"gorm.io/gorm"
-	"plugin"
 )
 
 type Deps struct {
 	SourceDB            *gorm.DB
+	SourceDBSchema      string
```
Comment (chengchengpei) [inline utils/deps.go]: SourceDBSchema already in config

we do not need it

in fact, i plan to remove `DestinationDBSchema`

File: dao/beacon/attestations.gen.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// Code generated by gorm.io/gen. DO NOT EDIT.
+// Code generated by gorm.io/gen. DO NOT EDIT.
+// Code generated by gorm.io/gen. DO NOT EDIT.
+
+package beacon
```
Comment (chengchengpei) [inline dao/beacon/attestations.gen.go:5]: all testnets of beacon are the same schema?

File: dao/beacon/attestations.gen.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+const TableNameAttestation = "attestations"
+
+// Attestation mapped from table <attestations>
+type Attestation struct {
+	SlotNumber      int64     `gorm:"column:slot_number;primaryKey" json:"slot_number"`
+	Index           int32     `gorm:"column:index;primaryKey" json:"index"`
+	BeaconBlockRoot string    `gorm:"column:beacon_block_root" json:"beacon_block_root"`
+	AggregationBits string    `gorm:"column:aggregation_bits" json:"aggregation_bits"`
+	AttestationSlot int64     `gorm:"column:attestation_slot" json:"attestation_slot"`
```
Comment (chengchengpei) [inline dao/beacon/attestations.gen.go:19]: please double check the schema and data types...
i do not review them...

File: dao/beacon/attestations.gen.go (hunk 2 above)
Comment (mwrrlvkixj) [reply dao/beacon/attestations.gen.go:5]: yes

File: dao/beacon/attestations.gen.go (hunk 3 above)
Comment (mwrrlvkixj) [reply dao/beacon/attestations.gen.go:19]: sure

**Pull request merged**